### PR TITLE
Draft: Fix several translation issues

### PR DESCRIPTION
**Pull request by Roy-043**
PR to fix several translation issues. See forum topic.

Forum topic:
https://forum.freecadweb.org/viewtopic.php?f=23&t=65346

Additionally:
* Removed the faulty `_tr` and `_qtr` functions from `translate.py`. They are no longer used by the Draft or Arch WB.
* In `view_base.py`: Fixed wrong use of variables in `QT_TRANSLATE_NOOP` calls and changed the tooltips from 'hatch' pattern to 'SVG' pattern.

- [x]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [x]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- x]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [x]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the [FreeCAD bug tracker issue number](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) in case a particular commit solves or is related to an existing issue on the tracker. Ex: `Draft: fix typos - fixes #0004805`

---

### CODE REVIEW

File: src/Mod/Draft/draftguitools/gui_beziers.py [hunk 1]
```diff
@@ -60,14 +60,14 @@ def GetResources(self):
         """Set icon, menu and tooltip."""
 
         return {'Pixmap': 'Draft_BezCurve',
-                'Accel': "B, Z",
-                'MenuText': QT_TRANSLATE_NOOP("Draft_BezCurve", "Bezier curve"),
-                'ToolTip': QT_TRANSLATE_NOOP("Draft_BezCurve", "Creates an N-degree Bezier curve. The more points you pick, the higher the degree.\nCTRL to snap, SHIFT to constrain.")}
+                'Accel': 'B, Z',
+                'MenuText': QT_TRANSLATE_NOOP('Draft_BezCurve', 'Bézier curve'),
```
Comment (benj5378) [inline src/Mod/Draft/draftguitools/gui_beziers.py:64]: Why are the `"` changed to `'`? Standard should be " as in the rest of the file and PEP8

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5445">pipeline status</a> for feature branch [PR_5445](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5445). Pipeline [457078691](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/457078691) was triggered at [a70df35](https://github.com/FreeCAD/FreeCAD/pull/5445/commits/a70df35d12ca809045a5a6725e47b4821fd07289). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Draft/draftguitools/gui_beziers.py (hunk 1 above)
Comment (chennes) [reply src/Mod/Draft/draftguitools/gui_beziers.py:64]: @Roy-043 this appears to have happened at a few places in this changeset -- can you revise to use double quotes?

Comment (Roy-043) [reply src/Mod/Draft/draftguitools/gui_beziers.py:64]: If you look carefully you will see that in the original file a mix on single and double quotes are used while defining the same dictionary. So there is no 'standard' in the file. Also: the original file was recently refactored.

But I'll fix this anyway.

Comment (Roy-043) [reply src/Mod/Draft/draftguitools/gui_beziers.py]: Hmm, actually there was no need to fix this:
https://forum.freecadweb.org/viewtopic.php?f=8&t=65753

@benj5378 Can you confirm please.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5445">pipeline status</a> for feature branch [PR_5445](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5445). Pipeline [458496873](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/458496873) was triggered at [58360bd](https://github.com/FreeCAD/FreeCAD/pull/5445/commits/58360bd02706e7941be909cb2e23b84fd497cdc6). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Draft/draftguitools/gui_beziers.py (hunk 1 above)
Comment (benj5378) [reply src/Mod/Draft/draftguitools/gui_beziers.py]: I don't see any specific conclusion in the forum thread? They mention some personal exceptions for when they wouldn't use PEP8.

Regarding Zolko's comment, I agree with him, but it's for a whole other case: naming of variables and functions. Not code formatting regarding use of symbols, spaces, line breaks etc.

I noticed that the file is mixed, but forward on, I'd personally prefer it was always changed towards PEP8.

Conclusion, do whatever you want and I'll make a seperate PR afterwards doing PEP8

Comment (Roy-043) [reply src/Mod/Draft/draftguitools/gui_beziers.py]: PEP8 allows both single and double quotes.
Link provided in 1st forum topic answer:
https://www.python.org/dev/peps/pep-0008/#string-quotes

So your initial comment here does not make sense.

> Standard should be " as in the rest of the file

> I noticed that the file is mixed

And with that you initial comment makes even less sense.

Striving for consistent use of double or single quotes in the Python code of FreeCAD is a lost cause if ever I saw one. But, enough said, let's merge!

Comment (chennes) [reply src/Mod/Draft/draftguitools/gui_beziers.py]: As you say 😎

**Pull request merged**